wp/wp-content/themes/lahma_maker/assets/playerfiles/radio_player.vue

```diff
@@ -245,6 +245,26 @@ export default {
         "show_subtitle": function() {
             return this.np.now_playing.song.title
         },
+        "is_OFF_AIR": function() {
+            if (this.np.now_playing.playlist == 'OFF AIR' || this.np.now_playing.playlist !== 'Off Air Ambient') {
```

Comment:
would use exact comparison `===` strange your IDE doesn't complain about this...
I see you corrected later the negation of the `Off Air Ambient`...